LEAF_Request_Portal/templates/reports/LEAF_import_data.tpl

```diff
@@ -139,7 +140,7 @@
 
     function checkFormatExisting(column) {
         for (var i = 1; i < sheet_data.cells.length; i++) {
-            var value = sheet_data.cells[i][column].toString();
+            var value = sheet_data.cells[i] !== undefined && sheet_data.cells[i][column] !== undefined ? sheet_data.cells[i][column].toString() : '';

```

Comment:
I personally have a preference for ```typeof sheet_data.cells[i] !== "undefined"``` which I feel is safer, but I don't believe this needs to be changed.

Reply:
Addressed in commit `bd5`